MediaBrowser.Providers/Lyric/TxtLyricParser.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.IO;
+using Jellyfin.Extensions;
+using MediaBrowser.Controller.Lyrics;
+using MediaBrowser.Controller.Resolvers;
+
+namespace MediaBrowser.Providers.Lyric;
+
+/// <summary>
+/// TXT Lyric Parser.
+/// </summary>
+public class TxtLyricParser : ILyricParser
+{
+    private static readonly string[] _supportedMediaTypes = { ".lrc", ".elrc", ".txt" };
+    private static readonly string[] _lineBreakCharacters = { "\r\n", "\r", "\n" };
+
+    /// <inheritdoc />
+    public string Name => "TxtLyricProvider";
+
+    /// <summary>
+    /// Gets the priority.
+    /// </summary>
+    /// <value>The priority.</value>
+    public ResolverPriority Priority => ResolverPriority.Fifth;
+
+    /// <inheritdoc />
+    public LyricResponse? ParseLyrics(LyricFile lyrics)
+    {
+        if (!_supportedMediaTypes.Contains(Path.GetExtension(lyrics.Name.AsSpan()), StringComparison.OrdinalIgnoreCase))
+        {
+            return null;
+        }
+
+        string[] lyricTextLines = lyrics.Content.Split(_lineBreakCharacters, StringSplitOptions.None);
+
+        if (lyricTextLines.Length == 0)
```

Comment:
This can't happen. It will always contain at least one element (unless the string is null, which results in NullReferenceException). You should check whether the string is null or empty prior to the split.